test/test_migrate.py

```diff
@@ -137,3 +137,25 @@ def test_pacman():
     """)
 
     check_migrate(orig, expected)
+
+
+def test_new_at_current_to_new_client_position():
+    orig = textwrap.dedent("""
+        from libqtile import layout
+
+        layouts = [
+            layout.MonadTall(border_focus='#ff0000', new_at_current=False),
+            layout.MonadWide(new_at_current=True, border_focus='#ff0000'),
+        ]
+    """)
+
+    expected = textwrap.dedent("""
+        from libqtile import layout
+
+        layouts = [
+            layout.MonadTall(border_focus='#ff0000', new_client_position=after_current),
```

Comment:
Just reading this test -- shouldn't "after_current" be a string?

Reply:
Outch! 
You are right, the migration will certainly generate a non loadable config without adding quote around the value of the key.
I will push a new commit modifying the migration and the test.